demo/csi/nfs/setup.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# Set up all the demo components.
+# This can be run repeatedly as it is fairly idempotent.
+
+set -xeuo pipefail
```

Comment:
Do we want `-x` still or was that from debugging?

Reply:
Personally I like `-x` in general, but especially for demos I like to be transparent about what's actually being done, even if the script is right there to be read.  Sometimes it's a bit annoying, like with the while loops polling until next step, but overall I'm a fan.

Reply:
Works for me!